prod-config.yml

```diff
@@ -90,22 +90,6 @@ metadata:
 europeanIdentity:
   enabled: false
 
-  # Details about eIDAS metadata can be configured here
```

Comment:
~How does the config class handle values that are supplied in the config file that don't exist as a field? I'm guessing that the services will try to spin up the MSA with their old config file which will include these keys. Are they going to have to remove them?~
I tested this - it doesn't care about extra keys